Aztec/Classes/Libxml2/DOMString.swift

```diff
@@ -341,6 +331,11 @@ extension Libxml2 {
         
         // Apply Styles
         
+        func apply(element: StandardElementType, spanning range: NSRange, attributes: [Attribute] = []) {
```

Comment:
This should really be private, as the mapping between styles and DOM elements should be handled by `DOMString` itself.  Read my comments below.